content/docs/deployment/csmoperator/_index.md

```diff
@@ -86,6 +86,36 @@ The `Update approval` (**`InstallPlan`** in OLM terms) strategy plays a role whi
 
 **NOTE**: The recommended version of OLM for Upstream Kubernetes is **`v0.18.3`**.
 
+### Upgrade driver using Dell CSM Operator:
+The CSI Drivers installed by the Dell CSM Operator can be updated like any Kubernetes resource.
+* Modifying the installation directly via `kubectl edit`
+    ```
+    $ kubectl get <driver-object> -n <driver-namespace>
+    ```
+    For example - If the CSI PowerFlex driver is installed then run this command to get the object name
+    ```
+    # Replace driver-namespace with the namespace where the CSI PowerFlex driver is installed
```

Comment:
Could we use the example of PowerStore driver instead of Powerflex. We are considering to change from vxflexos to PowerFlex for CSI PowerFlex Driver. So it would be good to use PowerStore as an example

Reply:
sure

Reply:
Done